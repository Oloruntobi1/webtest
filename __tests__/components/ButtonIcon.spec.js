import React from 'react';
import renderer from 'react-test-renderer';
import ButtonIcon from '../../components/Common/Button/Icon';

it('renders correctly', () => {
  const tree = renderer
    .create(<ButtonIcon src="/assets/svg/icons/phone.svg" name="phone" />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
