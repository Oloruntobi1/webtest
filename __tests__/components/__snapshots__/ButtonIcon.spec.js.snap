// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<button
  className="btn transition-3d-hover bg-white d-flex align-items-center justify-content-center border rounded-circle"
  style={
    Object {
      "height": "45px",
      "width": "45px",
    }
  }
  type="button"
>
  <img
    alt="phone"
    height={20}
    src="/assets/svg/icons/phone.svg"
    width={20}
  />
</button>
`;
